### ISSUE TITLE
CI: graphical glithes on hw12 worker (Framework 13 AMD)

### ISSUE THREAD

**Issue by marmarek** [QubesOS/qubes-issues#9035]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release

R4.2

### Brief summary

There are a lot of glitches on the HDMI output as captured by the RPi. They are not preventing openQA from working (there are still frames that are clean), but it's very annoying to work with.

### Steps to reproduce

Connect to hw12 via VNC, or watch it on openQA-captured video.

### Expected behavior

Clean image of HDMI output

### Actual behavior

https://openqa.qubes-os.org/tests/93628#step/update2/32
(see video too)

This is likely an issue with RPi capturing the video, maybe some cable is broken (broken shielding or such). But it could be also an issue with the "laptop" itself too (it's a Framework 13 AMD motherboard in the standalone mode). I did not had this issue when connected to a laptop screen.

Interestingly, glitches intensify greatly when there is activity on the wifi card (like downloading updates). Normally it's just flashing horizontal bands, but during network activity, most of the screen is garbled.

**Comment by marmarek** [comment]
One thing to start with is to connect HDMI to something else (a monitor) and see if that still happens

Another debugging idea is to decode pixel data of glitched screen. If it isn't random, but some structured data, it may suggest from where it comes.

**Comment by DemiMarie** [comment]
I have the same problem with the hw14 worker (System76).

**Comment by marmarek** [comment]
is it the same pattern as in the linked screenshot?

**Comment by DemiMarie** [comment]
No.  The screen flashes green, and sometimes the connection drops with `ECONNRESET`.

**Comment by marmarek** [comment]
This is most likely some issue on the RPi side, related to wifi. When I disable hotspot on the RPi (stop hostapd service), image is perfectly stable. Even if the laptop is connected to another wifi and is downloading stuff. On the other hand, if wifi on the laptop is disabled completely, but hostapd service is running on the RPi, glitches are back.

**Comment by DemiMarie** [comment]
That seems like RF interference to me.  Can you connect RPi to wired Ethernet instead?

**Comment by marmarek** [comment]
That's not the problem, RPi _is_ connected by wired eth. But it also hosts an AP for the test laptop to test wifi (and in case of Framework, serve as its main network connection, because Framework doesn't have onboard ethernet at all...).
Anyway, this is probably some issue with this particular piece, I don't have such issues on any other.
